scripts/bundle_bundles.py

```diff
@@ -11,7 +11,7 @@
 from _common import TEMPLATE_WORKSPACE
 
 DEFAULT_ZIP_FILENAME = "custom-channel-ui.zip"
-SKIP_WORKSPACES = ["kolibri-api", TEMPLATE_WORKSPACE]
+SKIP_WORKSPACES = ["kolibri-api", "loading-screen", TEMPLATE_WORKSPACE]
```

Comment:
I ran `lerna list --json --loglevel error` and it seems all 3 results will be skipped now. So we can remove this bundling altogether.

```
[
  {
    "name": "kolibri-api",
    "version": "0.1.0",
    "private": false,
    "location": "/sysroot/home/manuq/eos/hack/gitrepos/kolibri-explore-plugin/packages/kolibri-api"
  },
  {
    "name": "loading-screen",
    "version": "0.1.0",
    "private": false,
    "location": "/sysroot/home/manuq/eos/hack/gitrepos/kolibri-explore-plugin/packages/loading-screen"
  },
  {
    "name": "template-ui",
    "version": "0.1.0",
    "private": false,
    "location": "/sysroot/home/manuq/eos/hack/gitrepos/kolibri-explore-plugin/packages/template-ui"
  }
]
```